Configuration/Skimming/src/DisappearingMuonsSkimming.cc

```diff
@@ -0,0 +1,444 @@
+    if (ROOT::Math::VectorUtil::DeltaR(idPositionRoot, trajRoot) < ecalIsoConesize_ && (*hit).energy() > minEcalHitE_) {
+      eDR += (*hit).energy();
+    }
+  }
+  for (EcalRecHitCollection::const_iterator hit = rechitsEB->begin(); hit != rechitsEB->end(); hit++) {
+    const DetId id = (*hit).detid();
+    const GlobalPoint hitPos = caloGeom.getSubdetectorGeometry(id)->getGeometry(id)->getPosition();
+    if ((hitPos.eta() * t0.momentum().eta()) < 0) {
+      continue;
+    }
+    TrajectoryStateClosestToPoint traj = track.trajectoryStateClosestToPoint(hitPos);
+    math::XYZVector idPositionRoot(hitPos.x(), hitPos.y(), hitPos.z());
+    math::XYZVector trajRoot(traj.position().x(), traj.position().y(), traj.position().z());
+    if (ROOT::Math::VectorUtil::DeltaR(idPositionRoot, trajRoot) < ecalIsoConesize_ && (*hit).energy() > minEcalHitE_) {
+      eDR += (*hit).energy();
+    }
+  }
+
+  return eDR;
+}
+
+void DisappearingMuonsSkimming::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  desc.add<edm::InputTag>("recoMuons", edm::InputTag("muons"));
+  desc.add<edm::InputTag>("tracks", edm::InputTag("generalTracks"));
+  desc.add<edm::InputTag>("StandaloneTracks", edm::InputTag("standAloneMuons"));
+  desc.add<edm::InputTag>("primaryVertices", edm::InputTag("offlinePrimaryVertices"));
+  desc.add<edm::InputTag>("EERecHits", edm::InputTag("reducedEcalRecHitsEE"));
+  desc.add<edm::InputTag>("EBRecHits", edm::InputTag("reducedEcalRecHitsEB"));
+  desc.add<edm::InputTag>("TriggerResultsTag", edm::InputTag("TriggerResults", "", "HLT"));
+  desc.add<std::vector<std::string>>("muonPathsToPass",
+                                     {
+                                         "HLT_IsoMu24_v",
+                                         "HLT_IsoMu27_v",
+                                     });
+  desc.add<double>("minMuPt", 26);
+  desc.add<double>("maxMuEta", 2.4);
+  desc.add<double>("minTrackEta", 0);
+  desc.add<double>("maxTrackEta", 2.4);
+  desc.add<double>("minTrackPt", 20);
+  desc.add<double>("maxTransDCA", 0.005);
+  desc.add<double>("maxLongDCA", 0.05);
+  desc.add<double>("maxVtxChi", 3.0);
+  desc.add<double>("minInvMass", 50);
+  desc.add<double>("maxInvMass", 150);
+  desc.add<double>("trackIsoConesize", 0.3);
+  desc.add<double>("trackIsoInnerCone", 0.01);
+  desc.add<double>("ecalIsoConesize", 0.4);
+  desc.add<double>("minEcalHitE", 0.3);
+  desc.add<double>("maxTrackIso", 0.05);
+  desc.add<double>("maxEcalIso", 10);
+  desc.add<double>("minSigInvMass", 76);
+  desc.add<double>("maxSigInvMass", 106);
+  desc.add<double>("minStandaloneDr", 1.0);
+  desc.add<double>("maxStandaloneDE", -0.5);
+  desc.add<bool>("keepOffPeak", true);
+  desc.add<bool>("keepSameSign", true);
+  desc.add<bool>("keepTotalRegion", true);
+  desc.add<bool>("keepPartialRegion", true);
+  descriptions.add("disappMuonsSelection", desc);
```

Comment:
```suggestion
  descriptions.addWithDefaultLabel(desc);
```
This creates the file cfipython/Configuration/Skimming/disappearingMuonsSkimming_cfi.py

Reply:
Thanks, I put the in addWithDefaultLabel and Configuration.Skimming.disappearingMuonsSkimming_cfi import and verified that the cfi file is produced and cmsDriver successfully generates a config file for the skim.